cfg/pgrapher/experiment/dune10kt-1x2x6/wcls-sim-drift-simchannel.jsonnet

```diff
@@ -26,6 +26,7 @@ local params = base {
   },
 };
 
+local G4RefTime = std.extVar('G4RefTime') * wc.us;
```

Comment:
Is it possible to also upload the LArSoft fhicl or just document the fhicl workflow in a comment? It would be easier for other people to run these configurations.

Reply:
You mean this [fcl](https://github.com/WireCell/wire-cell-toolkit/pull/95/files#diff-7b225674e345dae7f70aa820a16d8cfcfea4d6ffc7ee981c56f2b356ed7af0a6R51) file?